### DISCUSSION TITLE
How to set up Microsoft authentication for your website

### DISCUSSION THREAD

**Discussion by Seizal** [microsoftgraph/msgraph-sdk-go#478] [Q&A]
Hello, 

I'm a young developer who's looking to set up Microsoft authentication in my GO-based web service. Initially, I was able to retrieve the token in version 0.58.0, but now I would like to do it in the latest version. However, it seems that some functions have changed names. Do you have any advice to give me?

Thank you for you time.

**Comment by baywet** [comment]
Can you provide a code snippet of the code you had with 0.58 ?
